bugbug/model.py

```diff
@@ -356,8 +357,10 @@ def train(self, importance_cutoff=0.15, limit=None):
         is_multilabel = isinstance(y[0], np.ndarray)
         is_binary = len(self.class_names) == 2
 
-        # Split dataset in training and test.
-        X_train, X_test, y_train, y_test = self.train_test_split(X, y)
```

Comment:
We should keep using `self.train_test_split()` - it is overridden in some models. 

You could create a similar method and call it `train_validation_split`.

Reply:
so I should use self.train_test_split() to split the data or I should create another method train_validation_split to split the data?

Reply:
Also the self.train_test_split() doesnt a parameter to specify the size of splitting

Reply:
A better solution imo can be to use Fast_ml's ‘train_valid_test_split’ , it splits the data into train/validation/test in a single line of code

Reply:
You can create a method called `train_validation_split`; then give it the training data from `self.train_test_split()`:

```py
X_train, X_test, y_train, y_test = self.train_test_split(X, y)
X_train, X_val, y_train, y_val = self.train_validation_split(X_train, y_train)
```